Kitodo/src/main/java/de/sub/goobi/helper/tasks/CreatePdfFromServletThread.java

```diff
@@ -150,7 +150,7 @@ public void run() {
                 logger.error("URI " + this.targetFolder + this.getProcess().getTitle() + ".PDF-ERROR.log is malformed",
                     e1);
             } catch (IOException e1) {
-                logger.error("Ressource " + uri + " could not be created", e);
+                logger.error("Resource could not be created", e);
```

Comment:
Why did you remove the name / content / value of not created URI? This could help on search for error reason.

Reply:
uri is null here

Reply:
I see it now and removing is ok in this case.